frontend/src/pages/cans/detail/BudgetSummary.jsx

```diff
@@ -1,12 +1,12 @@
 import { useSelector, useDispatch } from "react-redux";
-import { getCfyByCan } from "./getCfy";
+import { getCanFiscalYearByCan } from "./getCanFiscalYear";
 import { useEffect, useState } from "react";
 import Select from "react-select";
 import { useParams } from "react-router-dom";
 
 const BudgetSummary = () => {
     const dispatch = useDispatch();
-    const cfy = useSelector((state) => state.cfyDetail.cfy);
+    const can_fiscal_year = useSelector((state) => state.canFiscalYearDetail.canFiscalYear);
```

Comment:
"Snake case" isn't really used in JavaScript.  I'd rename this to `canFiscalYear`.

Reply:
fixed